Formula/libcbor.rb

```diff
@@ -13,6 +13,8 @@ class Libcbor < Formula
 
   depends_on "cmake" => :build
 
+  uses_from_macos "cjson" => :build
+
   def install
     mkdir "build" do
       system "cmake", "-G", "Unix Makefiles", "..", *std_cmake_args
```

Comment:
Try turning off IPO/LTO:
```suggestion
      system "cmake", "..", "-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF", *std_cmake_args
```